src/benchmarks/gc/docs/bench_file.md

```diff
@@ -7,8 +7,7 @@ Each is a map with keys being arbitrary names.
 Here's an example benchfile:
 
 ```yaml
-test_executables:
-  defgcperfsim: /performance/artifacts/bin/GCPerfSim/release/netcoreapp5.0/GCPerfSim.dll
+test_executables: {}
```

Comment:
what does {} mean here?

Reply:
Linter screwed up some things again when adding Andrew's encapsulation changes. I've converted the PR to draft to fix this and other things. I'll let you know once it's ready for review again.